bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/ESP32ClangCmakeToolChainProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.espressif.idf.core.build;
+
+import org.eclipse.cdt.core.build.IToolChain;
+import org.eclipse.core.runtime.CoreException;
+
+public class ESP32ClangCmakeToolChainProvider extends AbstractESPCMakeToolChainProvider
+{
+
+	public static final String TOOLCHAIN_NAME = "toolchain-clang-esp32.cmake"; //$NON-NLS-1$
+
+	@Override
+	protected IToolChain getToolchain() throws CoreException
+	{
+		return tcManager.getToolChain(ESPToolChainProvider.ID, ESP32ToolChain.ID);
+	}
```

Comment:
Here we need to keep ESP32ClangToolChain.ID?

Reply:
fixed